assets/src/edit-story/components/keyboard/index.js

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Mousetrap from 'mousetrap';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext, useEffect, createRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Context from './context';
+
+const PROP = '__WEB_STORIES_MT__';
+const NON_EDITABLE_INPUT_TYPES = ['submit', 'button', 'checkbox', 'radio'];
```

Comment:
```suggestion
const NON_EDITABLE_INPUT_TYPES = ['submit', 'button', 'checkbox', 'radio', 'image', 'file', 'range', 'reset', 'hidden'];
```

Though a focussed `hidden` would be weird 😮.

Reply:
Done.